run_analysis/run_analysis.R

```diff
@@ -304,7 +330,30 @@ run_analysis <- function(data, settings) {
       if (length(settings$compliance) < 8) {
         combined_data_f <- filter(combined_data_f, compliance_status %in% settings$compliance)
       }
-      
+
+      # -------- determine AS4777.2:2020 over-frequency f-W compliance --------
```

Comment:
This 2020 f-W is very nearly identical to the 2015, just with different column names.
It would be best to move this into a function that either is called twice with a param to tell it which to use, or that runs these sequentially.